contact.html

```diff
@@ -0,0 +1,613 @@
+                                            </div>
+
+                                            <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <div data-cmp-lazythreshold="0"
+                                                    data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg"
+                                                    data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg"
+                                                    data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image"
+                                                    itemscope="" itemtype="http://schema.org/ImageObject">
+                                                    <a href="#" class="cmp-image__link">
+                                                        <img src="./assets/linkedin.svg" class="cmp-image__image"
+                                                            itemprop="contentUrl" data-cmp-hook-image="image"
+                                                            alt="LinkedIn">
+                                                    </a>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+          </div>
+          <!-- CONTACT -->
+          <div class="container responsivegrid contact-us aem-GridColumn aem-GridColumn--default--12">
+            <div id="contact-container" class="cmp-container">
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+
+                    <div class="container responsivegrid contact-content aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div class="cmp-container">
+                            <div class="contact-title">
+                                <div id="title-fca834a3a8" class="cmp-title">
+                                    <h1 class="cmp-title__text">Contact Us</h1>
+                                </div>
+                            </div>
+                            <div class="contact-subtext text base">
+                                <div data-cmp-is="adaptiveFormText" id="text-53cc58dc93" name="text_demo" data-cmp-adaptiveformcontainer-path="/content/core-components-examples/library/adaptive-form/text/jcr:content/root/responsivegrid/demo/component/guideContainer" class="cmp-adaptiveform-text">
```

Comment:
@Prystal254 We don't have the adaptive form components. Of the six categories of components listed in the [documentation](https://www.aemcomponents.dev/content/core-components-examples/library.html), we only have the first three. Use a regular text component here instead.